runtime/parachains/src/inclusion.rs

```diff
@@ -592,7 +600,13 @@ impl<T: Config> Module<T> {
 		para_id: ParaId,
 		validation_outputs: primitives::v1::CandidateCommitments,
 	) -> bool {
-		if let Err(err) = CandidateCheckContext::<T>::new().check_validation_outputs(
+		// This function is meant to be called from the runtime APIs against the relay-parent, hence
```

Comment:
should we add `for_runtime_api` to the function name?

Reply:
My initial impression was good call and I did the change locally.

Then I've realized that:

- we have a single function in `runtime_api_impl` that has this distinction.
- in the chain of function calls starting from the runtime API, was looking 
`check_validation_outputs - check_validation_outputs - check_validation_outputs - check_validation_outputs` 
now it would look 
`check_validation_outputs - check_validation_outputs - check_validation_outputs_for_runtime_api - check_validation_outputs`
- this piece of code that looks (deliberately) weird, i.e. `let relay_parent_number = now;` is still there which means that we still need the comment here,
- the name became longer than I am typically comfort with

so now i am slightly inclined to say no, but I can push it nonetheless, since none of those are strong arguments

Reply:
Well, maybe a slightly different name change then, that is shorter? I would like to have the name reflect the internals better.

Reply:
k, I'll leave `for_runtime_api` then